### ISSUE TITLE
abstract-transport.js: callback is not a function

### ISSUE THREAD

**Issue by SirLancelot-OG** [AugurProject/augur.js#425]
`augur.connect` is throwing this error (stemming from [ethrpc](https://github.com/ethereumjs/ethrpc/blob/master/src/transport/abstract-transport.js))

```
abstract-transport.js:108
    callback(null, this);
    ^

TypeError: callback is not a function
    at WsTransport.<anonymous> (E:\MAMP_V3\htdocs\augur-api\node_modules\ethrpc\src\transport\abstract-transport.js:108:5)
    at callback (E:\MAMP_V3\htdocs\augur-api\node_modules\ethrpc\src\transport\ws-transport.js:21:5)
    at W3CWebSocket.webSocketClient.onopen (E:\MAMP_V3\htdocs\augur-api\node_modules\ethrpc\src\transport\ws-transport.js:27:33)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (E:\MAMP_V3\htdocs\augur-api\node_modules\yaeti\lib\EventTarget.js:117:12)
    at W3CWebSocket.onConnect (E:\MAMP_V3\htdocs\augur-api\node_modules\websocket\lib\W3CWebSocket.js:208:10)
    at WebSocketClient.<anonymous> (E:\MAMP_V3\htdocs\augur-api\node_modules\websocket\lib\W3CWebSocket.js:55:19)
    at WebSocketClient.emit (events.js:159:13)
    at WebSocketClient.succeedHandshake (E:\MAMP_V3\htdocs\augur-api\node_modules\websocket\lib\WebSocketClient.js:335:10)
    at WebSocketClient.validateHandshake (E:\MAMP_V3\htdocs\augur-api\node_modules\websocket\lib\WebSocketClient.js:319:10)
    at ClientRequest.handleRequestUpgrade (E:\MAMP_V3\htdocs\augur-api\node_modules\websocket\lib\WebSocketClient.js:248:14)
```

# Additional info
Using the following `http` and `ws` nodes

**Ethereum**
```
var ethereumNode = { 
  httpAddresses: [
    "rinkeby.ethereum.nodes.augur.net"
  ],
  wsAddresses: [
    "wss://websocket-rinkeby.ethereum.nodes.augur.net"
  ]
};
```

**Augur**
`var augurNode = "wss://rinkeby.augur.nodes.augur.net";`

**Comment by jorgeolivero** [comment]
@SirLancelot-OG 

@epheph fixed with https://github.com/AugurProject/augur.js/pull/432

**Comment by epheph** [comment]
Yes, i changed the WsTransport constructor params inside ethrpc without realizing augur.js re-purposed WsTransport for its (non-eth) connection to augur-node

**Comment by SirLancelot-OG** [comment]
Issue was fixed with #432
